style.css

```
@@ -81,3 +87,54 @@ body {
 .btn_menu:hover {
 	background-color: #3AC162;
 }
+
+/*Welcome*/
+
+.container {
+	background-image: url('images/overlay.png'), url('images/hero-bg.jpg');
+	background-size: cover;
+	background-repeat: no-repeat;
+	background-attachment: fixed;
+	min-height: 500px;
+  padding-top: 180px;
+  background-position: center top;
+}
+
+.welcome_container {
+	max-width: 1150px;
+	padding: 0 10px 0;
```

Já que o valor do top e do bottom são iguais, dá pra simplificar:
```suggestion
	padding: 0 10px;
```